build_stats_functions/README.md

```diff
@@ -0,0 +1,18 @@
+## Build Stats Functions
+
+Python Files that analyze and parse Bazel Build statistics.
+
+Maintainer: @halseycamilla (Tensorflow)
+
+* * *
+
+Python Google Cloud Functions that use EventArc triggers to grab and then parse build profiles and build event protocols from TensorFlow's public build logs. The functions are called every time a build log is uploaded to the Google Cloud Storage bucket storing TensorFlow's build logs. Once the function parses the data it uploads it to a Bigquery table. The folders also include a requirements.txt that states the necessary imports to run the functions.
+
+## Testing Locally:
+
+- Download any missing requirements from `requirements.txt` with `pip3 install`. Make sure Python 3.10 is currently installed before doing so.
```

Comment:
Nit: convert this to a numbered list, not a bullet-point list. It's an ordered sequence so that should be easier to understand.